CONTRIBUTING.md

```diff
@@ -233,6 +233,7 @@ If you have not done so on this machine, you need to:
     * macOS: Use the `Disk Utility.app` to check. It also allows you to create a case-sensitive volume to store your code projects. See this [blog entry](https://karnsonline.com/case-sensitive-apfs/) for more.
     * Windows: [Enable case sensitive file names per directory](https://learn.microsoft.com/en-us/windows/wsl/case-sensitivity)
 * Install Git and configure your GitHub access
+  * Windows: enable longpaths: `git config --global core.longpaths true`
```

Comment:
I'd also add the following, to avoid using the Windows CRLF breaks 
```bash
git config --global core.autocrlf false
```

Reply:
done